CHANGELOG.md

```diff
@@ -17,6 +17,24 @@ Features, Bug Fixes, Breaking Changes, Deprecated
 
 ### API breaking
 
+- We have updated the default behaviour of algob deployer for loading data from checkpoint to be queried by  "app/lsig" name (note: passing name is required). The existing functionality has been moved to `<func>ByFile` functions (legacy functions based on file querying):
+
+  - Application:
+    * Pervious `getApp(approval.py, clear.py)` has been changed to `getAppByFile(approval.py, clear.py)`.
+    * New `getApp(appName)` function queries app info using the app name.
+
+  - Smart signatures:
+    * Exisiting `getDelegatedLsig(lsig.py)`, `getContractLsig(lsig.py)` **have been removed**. Use `getLsig` funtion to query logic signature from name or filename in a checkpoint.
+    * New `getApp(appName)` function queries app info using the app name.
+    * Existing `fundLsig(lsig.py, ..)` function has been changed to `fundLsigByFile(lsig.py, ..)`. Now `fundLsig(lsigName, ..)` will take lsig name.
+    * Existing `mkDelegatedLsig(fileName, signer, ..)`, `mkContractLsig(fileName, ..)` have been updated to take the lsigName as a required paramter:
+      - `mkDelegatedLsig(lsigName, fileName, signer)`
+      - `mkContractLsig(lsigName, fileName)`.
+      Here `fileName` represent the name of smart contract file (eg. `treasury-lsig.teal`), and `lsigName` represents the "name" you want to assign to this lsig (eg. `treasuryLsig`).
+
+  For reference you can check out `examples/asa`.
```

Comment:
let's update the change log that the order of function arguments changed (list functions, and mention in bold that now "name" is a first argument and it's required).

Reply:
ok